uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -88,7 +95,8 @@ define([
          */
         defaultCustomData: function () {
           return {
-            comments: []
+            comments: [],
+            commentsToBeDeleted: []
```

Comment:
can we have attribute in comments array itself that will mark it for deletion like `canDelete`?